eng/tools/npm-admin-tasks.ps1

```diff
@@ -0,0 +1,60 @@
+param (
+  [Parameter(mandatory = $true)]
+  $taskType,
+  [Parameter(mandatory = $true)]
+  $packageName,
+  [Parameter(mandatory = $true)]
+  $pkgVersion,
+  $tagName,
+  [Parameter(mandatory = $true)]
+  $npmToken,
+  $reason
+)
+
+try {
+  Write-Host "Setting AuthToken Deployment"
+  $env:NPM_TOKEN = $npmToken
+  $regAuth = "//registry.npmjs.org/"
+  npm config set $regAuth`:_authToken=`$`{NPM_TOKEN`}
+  $nameAndVersion = $packageName + "@" + $pkgVersion
+
+  # Verify that package name is not "@azure"
+  if ($packageName -eq '@azure') {
+    Write-Host "Invalid package name"
+    exit 1
+  }
+
+  switch ($taskType) {
+    AddTag {
```

Comment:
Does this need to be surrounded in quotes?

Reply:
It is working when I tested as script and through pipeline. Looking at powershell examples, It doesn't seems necessary. If you think it's better to surround as quoted string, I can make that change.

Reply:
I'm jus surprised it actually works because generally PS doesn't automatically covert things to strings like this. When I saw this I thought they were enum values somewhere. I'd suggest making them quoted so it is clear they are strings.